vspd.go

```diff
@@ -37,6 +38,13 @@ func main() {
 	}
 }
 
+//Implementing password hash to increase security for AdminPass
+//hashPassword hash cfg.AdminPass and returns the hash.
+func hashPassword(password string) (string, error) {
```

Comment:
Not sure what the first line is for, but function comments should begin with the function name and describe what the function does. That seems to be done in the second line, so looks like the first comment line is unnecessary. Also note the space between the slash and the start of the comment.

```suggestion
// hashPassword hash cfg.AdminPass and returns the hash.
func hashPassword(password string) (string, error) {
```